openpype/hosts/maya/tools/mayalookassigner/app.py

```diff
@@ -240,18 +241,37 @@ def on_process_selected(self):
             ))
             nodes = item["nodes"]
 
+            # Assign Vray Proxy look.
             if cmds.pluginInfo('vrayformaya', query=True, loaded=True):
                 self.echo("Getting vray proxy nodes ...")
                 vray_proxies = set(cmds.ls(type="VRayProxy", long=True))
 
-                if vray_proxies:
-                    for vp in vray_proxies:
-                        if vp in nodes:
-                            vrayproxy_assign_look(vp, subset_name)
+                for vp in vray_proxies:
+                    if vp in nodes:
+                        vrayproxy_assign_look(vp, subset_name)
 
-                    nodes = list(set(item["nodes"]).difference(vray_proxies))
+                nodes = list(set(item["nodes"]).difference(vray_proxies))
+            else:
+                self.echo(
+                    "Could not assign to VRayProxy because vrayformaya plugin "
+                    "is not loaded."
+                )
+
+            # Assign Arnold Standin look.
+            if cmds.pluginInfo("mtoa", query=True, loaded=True):
+                arnold_standins = set(cmds.ls(type="aiStandIn", long=True))
```

Comment:
This doesn't necessarily need to be a set. We're doing no lookups here.
And `.difference()` call down below can work against any iterable as far as I know.

Reply:
I was following the VrayProxy code [here](https://github.com/tokejepsen/pype/blob/feature/OP-4859_cant-assign-shaders-to-the-ass-file/openpype/hosts/maya/tools/mayalookassigner/app.py#L247). If there is no issues (?) then I would just keep it as is.